modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -2715,14 +2717,18 @@ func (suite *KeeperTestSuite) TestWriteErrorReceipt() {
 				ch := path.EndpointA.GetChannel()
 				upgradeError = types.NewUpgradeError(ch.UpgradeSequence, types.ErrInvalidUpgrade)
 			},
-			errorsmod.Wrap(types.ErrInvalidUpgradeSequence, "attempting to write error receipt at sequence (1) while upgrade information exists at the same sequence"),
+			func() error {
+				return errorsmod.Wrap(types.ErrInvalidUpgradeSequence, "attempting to write error receipt at sequence (1) while upgrade information exists at the same sequence")
+			},
 		},
 		{
 			"failure: channel not found",
 			func() {
 				suite.chainA.DeleteKey(host.ChannelKey(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID))
 			},
-			errorsmod.Wrap(types.ErrChannelNotFound, "port ID (mock) channel ID (channel-0)"),
+			func() error {
+				return errorsmod.Wrap(types.ErrChannelNotFound, fmt.Sprintf("port ID (mock) channel ID (%s)", path.EndpointA.ChannelID))
+			},
```

Comment:
I don't particularly like changing this to a function, but I needed a way to be able to access `path.EndpointA.ChannelID` and in this way the function gets evaluated after `path.setup` has been called